test/prompConfig.js

```diff
@@ -1,11 +1,32 @@
+import fs from 'fs-extra';
 import test from 'ava';
 import path from 'path';
 import chalk from 'chalk';
 import json from 'json-extra';
+import os from 'os';
 import getConfig from '../lib/getConfig';
 import { choices, questions } from '../lib/promptConfig';
 
 const cwd = process.cwd();
+const homedir = os.homedir();
+const randomString = Math.random().toString(36).replace(/[^a-z]+/g, '').substr(0, 10);
```

Comment:
The benefit is that sometimes a `.back` file already exist. So if one exist we would overwrite the existing one. With the random string we ensure that there is no such file and we are good to go ✌️